### PR TITLE
Concent Terms of Service

### LINKED ISSUE #3550: [Concent] Create an RPC call to retrieve the Concent Terms of Service
Similar to the endpoint that provides Golem's TOS right now

Once created, please update the document here: https://github.com/golemfactory/golem/issues/3514

### LINKED ISSUE #3551: [Concent] Create the RPC to accept Concent ToS
analogous to the current "accept terms" call for Golem

Once created, please update the document here: https://github.com/golemfactory/golem/issues/3514

### PR DESCRIPTION
**Pull request by jiivan**
fixes: #3550 
fixes: #3551

### CODE REVIEW

File: golemapp.py [hunk 1]
```diff
@@ -84,6 +84,9 @@ def monkey_patched_getLogger(*args, **kwargs):
 @click.option('--accept-terms', is_flag=True, default=False,
               help="Accept Golem terms of use. This is equivalent to calling "
                    "`golemcli terms accept`")
+@click.option('--accept-concent-terms', is_flag=True, default=False,
+              help="Accept Concent terms of use. This is equivalent to calling "
+                   "`golemcli concent terms accept`")
```
Comment (Wiezzel) [inline golemapp.py:132]: Maybe just keep one option to accept all terms?

**Review by Wiezzel — APPROVED** [review]
Looks legit. I only have one suggestion.

File: golem/CONCENT_TERMS.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<h5><u>Disclaimer</u></h5>
+
+<span>
+This Disclaimer hasn't been created.
```
Comment (kmazurek) [inline golem/CONCENT_TERMS.html:4]: I think the terms themselves should be a part of this PR

File: golem/terms.py [hunk 3]
```diff
@@ -4,26 +4,38 @@
...
+
+class TermsOfUse(TermsOfUseBase):
+    ACCEPTED_KEY = 'terms_of_use_accepted'
+    VERSION = 3
+    PATH = Path('golem/TERMS.html')
+
+
+class ConcentTermsOfUse(TermsOfUseBase):
+    ACCEPTED_KEY = 'concent_terms_of_use_accepted'
+    VERSION = 0
```
Comment (kmazurek) [inline golem/terms.py:40]: Reminder: update version here once terms are added

File: tests/golem/test_terms.py [hunk 4]
```diff
@@ -1,57 +1,66 @@
...
+from golem import terms
 from golem.tools.testwithdatabase import TestWithDatabase
 
 
-class TestTermsOfUse(TestWithDatabase):
+fake = Faker()
+
+
+class TestTermsOfUseBase(TestWithDatabase):
+    def setUp(self):
```
Comment (kmazurek) [inline tests/golem/test_terms.py:15]: Is this override necessary?

File: golemapp.py (hunk 1 above)
Comment (kmazurek) [reply golemapp.py:132]: +1, or perhaps create a separate flag to accept all terms (for potential additions in the future)?

Comment (jiivan) [reply golemapp.py:132]: Hmm, it's perfectly possible to accept only golem terms without concent. I will add `--accept-all-terms`

File: tests/golem/test_terms.py (hunk 4 above)
Comment (jiivan) [reply tests/golem/test_terms.py:15]: Yes, to define `self.terms`

File: golem/CONCENT_TERMS.html (hunk 2 above)
Comment (kmazurek) [reply golem/CONCENT_TERMS.html:4]: Clarified: these can be added later on

**Review by kmazurek — APPROVED** [review]

**Pull request merged**